website/static/js/file-browser.js

```diff
@@ -0,0 +1,284 @@
+/**
+ * Builds full page project browser
+ */
+'use strict';
+
+var Treebeard = require('treebeard');   // Uses treebeard, installed as plugin
+var $ = require('jquery');  // jQuery
+var m = require('mithril'); // exposes mithril methods, useful for redraw etc.
+var ProjectOrganizer = require('js/project-organizer');
+var $osf = require('js/osfHelpers');
+
+/**
+ *  Options for fileBrowser
+ */
+var defaults = {
```

Comment:
Are these used? 

Also it may be cleaner to put all FileBrowser specific values under the FileBrowser namespace.